src/components/site/header/header.test.js

```diff
@@ -13,47 +39,34 @@ test('Header renders without props', () => {
 });
 
 test('Header Logo alt text is Investigation Title', () => {
+  const headerKey = 'interface::header.title';
   // Arrange
   const { getByAltText, getByText, queryByText } = render(
     <Header logo="fake/image/path" investigationTitle={testTitle} />
   );
   const logoLink = getByAltText(testTitle);
   // Assert
-  expect(getByText(testTitle)).toBeInTheDocument();
+  expect(getByText(headerKey)).toBeInTheDocument();
   expect(queryByText(`${testTitle}: Page`)).toBe(null);
   expect(logoLink).toBeInTheDocument();
 });
 
-test('Header renders page number if provided', () => {
```

Comment:
removed this test because we can't test for the page numbers without rendering the key in some way